src/App.tsx

```diff
@@ -1,13 +1,49 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+// import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { ToodoList } from './components/TodoList';
+import { FilterOptions } from './utils/FilterOptions';
+import cn from 'classnames';
+
+function getFilteredTodos(todos: Todo[], filterOption: FilterOptions): Todo[] {
```

Comment:
It is better to put the helper functions in a separate directory, all the more it already exists in our project

Reply:
Good point 🔥 
Move it to `utils` directory